bugbot/rules/not_landed.py

```diff
@@ -349,7 +329,8 @@ def get_bugs(self, date="today", bug_ids=[]):
             nickname = d["nickname"]
 
             if not assignee:
-                assignee, nickname = nicknames[bugid]
+                assignee_id = bug_assignee_map[bugid]
+                assignee, nickname = bz_reviewers[assignee_id]
```

Comment:
Based on the last modification, `bz_reviewers` returns a dict with the user data, not a tuple.